### ISSUE TITLE
Fix sr-editor desktop file name

### ISSUE THREAD

**Issue by AsciiWolf** [flathub/org.tuxfamily.StuntRally#9]
The Stunt Rally Track Editor (sr-editor) desktop file is correctly installed, however it has a wrong file name (sr-editor.desktop).
Adding something like this to _post-install_ should fix it and make Stunt Rally Track Editor appear in desktop menu:
```
"mv /app/share/applications/sr-editor.desktop /app/share/applications/org.tuxfamily.StuntRally.Editor.desktop"
```

**Comment by AsciiWolf** [comment]
Hopefully fixed in #7.

**Comment by AsciiWolf** [comment]
Also fixed in master.
